healthScore/tests.py

```diff
@@ -889,6 +889,17 @@ def test_approve_request(self):
         response = update_health_history_access_request_status(request)
         self.assertEqual(response.status_code, 200)
 
+    def test_reject_request(self):
```

Comment:
Could you also please write a test case for the email sent case?  Since this is a mail sending feature, it would be great to have a dedicated test case for it